package.json

```diff
@@ -615,7 +615,8 @@
 	"scripts": {
 		"compile": "tsc -p ./",
 		"watch": "tsc -p -watch ./",
-		"test": "npm run compile && node ./out/test/runTest.js"
+		"test": "npm run compile && node ./out/test/runTest.js",
+		"codecoverage": "npm run compile && node ./out/test/runCodeCoverage.js"
```

Comment:
thanks a lot! is this file uploaded?

Reply:
btw, I see even with code coverage, the test still runs very quickly. So this is not high priority as of now. We can just run the tests with coverage support.

Reply:
Yes, we can run the tests with coverage with this PR, and I will submit a new PR to decouple the code coverage to a separate command.

Reply:
I will remove this line from this PR